Chapters/2-2-Chapter9.md

```diff
@@ -0,0 +1,169 @@
+
+The woman whispered something in hae ear, and the maman looked Kipnu up and down with a mischievous smile. Thi motioned to him to join them on the other side of the cloth wall and she pushed him to go. With some hesitation, he followed. To his surprise, the other side wasn't the outside, but another smaller canopy with soft linens across the floor. He knew where this was going.
+
+The maman spoke to him with a charming face.
+
+"Aey Kipnu. I see my faithful hunter, Setre, has found you. Welcome to the Great Pavilion of Ulukoma. My name is Shinna. I own this island and all the people on it, including you. If you do what is asked of you, all of your dreams will be fulfilled," thi said, hae voice trailing off into some other time and place.
+
+"If you choose not to... well you've seen what happens to those who misbehave."
+
+Kipnu stared at hae with cautious eyes, wanting nothing more than to leave this place.
+
+Setre chimed in, "We hunt you down and kill you."
+
+Shinna laughed and continued, "Anyway, just cooperate and we'll all have a great time."
+
+Thi observed him with a feigned frown.
+
+"You seem a bit on-edge. Come. Share the spirit with us."
+
+Thi held up glowing blue syringe to Kipnu and encouraged him to take it. He took the device, familiar to him in touch but not in sight, and immediately tried to give it back.
+
+"See this is what I'm talking about," said Shinna, pulling out a dagger. Thi swiftly moved behind him and held it to his neck before he could move to escape. Realizing he wasn't going anywhere, he carefully looked closer at the syringe but an impatient Setre took it from him and held his arm tight enough to break it if he struggled any more.
+
+"Stop struggling!" she commanded, tightening her grip. Shinna was licking the rim of his ear while still holding a dagger to his neck.
+
+Setre moved the syringe to his abdomen, and punctured an artery, finishing the injection quicker than he realized. A blue glow began to radiate out from where the injection started, and immediately seized his lungs. He tried to cough without moving his neck into the dagger, and Shinna lowered it, saying "There you go, my little Heikoi doctor. You'll start to feel it."
+
+He continued to cough uncontrollably, struggling to regain control of his breathing. But then he felt his muscles relaxing, almost to the point of being immovable, and his head was rushing with countless thoughts and feelings; if he stopped to consider any of them, they would slip away in the next moment. His breathing became automatic. A passenger in his own body.
```

Comment:
Why is he coughing? It's not a substance he inhaled.

Reply:
I was thinking that it basically shuts off the frontal cortex's connection to the rest of the brain as one of its things.